### PR TITLE
Feature scriptified installer stacks

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by livecodeali** [comment]
Builds on https://github.com/runrev/livecode/pull/2054 but not quite working yet.

It seems there is an issue with including auxiliary stacks in the installer standalone.

**Comment by livecodeali** [comment]
Issue was caused by a couple of things already dealt with in revSaveAsStandalone stack, mode_installer.cpp in the engine, and some private functions and commands that I'd move to the installer utilities stack.

File: builder/release_notes_css_prefix.txt [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+
+@font-face{ font-family: MuseoSansForDell; font-style: normal; font-weight: normal; src: url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.eot), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.eot?#iefix) format("embedded-opentype"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.woff) format("woff"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.ttf) format("truetype"), url(http://livecode.com/wp-content/themes/livecode2013/fonts/museosansfordell-regular-webfont.svg#MuseoSansForDellRegular) format("svg"); }
+
```
Comment (livecodesebastien) [inline builder/release_notes_css_prefix.txt:550]: Could you update the `@font-face` to the lone
`@font-face{ font-family: FontAwesome; font-style: normal; font-weight: normal; src: url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.eot) format("embedded-opentype"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.woff) format("woff"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.ttf) format("truetype"), url(http://livecode.com/wp-content/themes/omega/assets/fonts/fontawesome-webfont.svg) format("svg"); }`

File: builder/release_notes_css_prefix.txt [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+/* ************************************************
+ * BASE STYLES
+ **************************************************/
+a{ text-decoration:none; color:#799327; } /* aecf36 */
+a:hover{ color:#585858}
+
+h1,h2,h3,h4{ font-family:MuseoSansForDellBold; font-style:bold; font-weight:bold}
```
Comment (livecodesebastien) [inline builder/release_notes_css_prefix.txt]: And update this to
`h1,h2,h3,h4{ font-family:FontAwesome; font-style:normal; font-weight:normal}`
That would bring in the fix brough in release-6.7.4, in develop-6.7

**Comment by livecodesebastien** [comment]
Is https://github.com/runrev/livecode/pull/2054 still valid, or should it be closed, as it seems to be included in this branch?

**Comment by livecodeali** [comment]
Thanks, I opened it in the interim while this branch was not working, but this one supersedes it now.

**Comment by livecodesebastien** [comment]
Thanks Ali.
Reviewed-by: Sébastien Nouat

**Pull request merged**